python/examples/tchannel_server.py

```diff
@@ -21,9 +21,7 @@
 from __future__ import absolute_import
 
 import tornado.ioloop
-
```

Comment:
Keep this. Whitespace between imports is used to distinguish between stdlib/third-party/local modules.